prow/getting_started.md

```diff
@@ -0,0 +1,81 @@
+# How to turn up a new cluster
+
+Prow should run anywhere that Kubernetes runs. Here are the steps required to
+set up a basic prow cluster on [GKE](https://cloud.google.com/container-engine/).
+You'll need a GCP project. Prow will work on any Kubernetes cluster, however,
+so feel free to turn up a cluster some other way and skip the first step.
+
+## Create the cluster.
+
+I'm assuming that `PROJECT` and `ZONE` environment variables are set. Run the
+following to create the cluster. This will also set up `kubectl` to point to
+the new cluster.
+
+```sh
+gcloud -q container --project "${PROJECT}" clusters create prow \
+  --zone "${ZONE}" --machine-type n1-standard-4 --num-nodes 2
+```
+
+## Create the GitHub secrets.
+
+You will need two secrets to talk to GitHub. The `hmac-token` is the token that
+you give to GitHub for validating webhooks. Generate it using any reasonable
+randomness-generator. The `oauth-token` is an OAuth2 token that has read and
```

Comment:
Can we recommend a reasonable randomness-generator?

Reply:
Done.